dev-docs/source/Testing/ErrorReporter-ProjectRecovery/ErrorReporterTesting.rst

```diff
@@ -0,0 +1,100 @@
+.. _error_reporter_testing:
+
+Error Reporter Testing
+======================
+
+.. contents::
+  :local:
+
+Error Reporter test
+-------------------
+
+*Preparation*
+
+-  Ensure that `Report usage data` is checked in the First Time Setup screen
+-  Files ``SANS2D00005512.nxs``
+-  Make sure the location of these files is included in your search
+   directories. Full instructions `are available
+   online <http://www.mantidproject.org/MBC_Getting_set_up#MantidPlot_First-Time_Setup>`__.
+- These tests should be run with someone who has access to the error reports database. Please contact Keith Butler when you are going to run these tests.
+
+**Time required 10 - 20  minutes**
+
+--------------
+
+1. Open MantidPlot
+
+- Cause a crash by running the `Segfault` algorithm
+- The error report dialog should pop up
+- Make sure the hyperlink to the privacy policy works
+- Enter some text in the `Name` box; make sure the `Share non-identifiable information` button gets greyed out
+- In the `Name` box enter `Private` and in the `Email` box enter `private`
+- Click the `Don't share any information` box
+
+---------------
+
+2. Open MantidPlot
+
+- Cause a crash by running the `Segfault` algorithm
+- The error report dialog should pop up
+- In the `Name` box enter `Public` and in the `Email` box enter `public`
+- Click the `Yes, share information` box
```

Comment:
Is it worth adding a step here and to the others to do with checking the message gets to the server as I think the tests are incomplete without this step even if it is not one which most people can do. It will at least let people know what messages we expect the server to receive in each case.